hw/ip/kmac/dv/kmac_masked_sim_cfg.hjson

```diff
@@ -0,0 +1,15 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// sim cfg file for the masked version of KMAC
+{
+  // Name of the sim cfg
+  name: kmac_masked
+
+  // Import additional common sim cfg files.
+  import_cfgs: ["{proj_root}/hw/ip/kmac/dv/kmac_base_sim_cfg.hjson"]
+
+  // Enable this build mode for all tests
+  en_build_modes: ["vcs_enable_mask_mode"]
```

Comment:
```suggestion
  en_build_modes: ["{tool}_enable_mask_mode"]
```

Reply:
I think we don't need this since we are now using a macro define instead of directly setting the parameter at compile time.